serenity-automation-mentoring/src/test/java/serenity/bdd/steps/DefinitionSteps.java

```diff
@@ -59,5 +58,24 @@ public void whenTheUserLooksUpTheDefinitionInTable(ExamplesTable wordTable) {
     public void thenTheyShouldSeeADefinitionContainingTheWordsInTable() {
        endUser.check_table();
     }
+//Pet
+    @Given("create pet with $name and $status")
+    public void createPetWithNameAndStatus(String name, String status) {
+        endUser.create_pet(endUser.getProperty(name), endUser.getProperty(status));
+    }
+
+    @When("find pet by status $status")
+    public void findPetByStatus(String status) {
+        endUser.find_by_status(status);
+    }
+
+    @When("delete pet by current id")
+    public void deletePetByCurrentId() {
+        endUser.delete_by_current_id();
+    }
 
+    @Then("check response code")
```

Comment:
Make this method parametrized, `check response code $code`

Reply:
Done